conn_test.go

```diff
@@ -70,12 +71,15 @@ func TestSSLSimple(t *testing.T) {
 
 func createTestSslCluster(hosts string, proto uint8) *ClusterConfig {
 	cluster := NewCluster(hosts)
-	cluster.SslOpts = &SslOptions{
-		CertPath:               "testdata/pki/gocql.crt",
-		KeyPath:                "testdata/pki/gocql.key",
+	sslOpts := &SslOptions{
 		CaPath:                 "testdata/pki/ca.crt",
 		EnableHostVerification: false,
 	}
+	if os.Getenv("REQ_CLIENT_AUTH") != "false" {
```

Comment:
That depends on whether we want to default to running with or without, right? Happy to change it, though.